imap.adoc

```diff
@@ -48,3 +48,12 @@ Do NOT configure a load balancer between the Zimbra HTTP Proxy node and any remo
 [IMPORTANT]
 The zmlocalconfig setting *nio_imap_enabled* MUST have value *true* when using the Zimbra IMAPD server.  The legacy
  Imap server which does not use NIO is not supported.
+
+==  IMAPD Configuration settings
+
+The following global and server level configuration attributes are available to control and tune the IMAPD service.
+
+* *zimbraRemoteImapServerEnabled*. When set to TRUE, the standalone IMAP server is enabled. When set to FALSE, the standalone IMAP server is disabled. Default value is FALSE.
```

Comment:
recommend using em dash ([space]--[space]) instead of period (.) after attribute names for each of these in the list

Reply:
I am going to leave this as-is at this time since the section above this also uses this same '.' separator.  Will be addressed in ZCS-4109